brokerapi/brokers/brokers_test.go

```diff
@@ -193,6 +193,13 @@ var _ = Describe("Brokers", func() {
 				}
 			}
 		})
+
+		It("should error if plan ids are not supplied", func() {
+			os.Setenv("PRECONFIGURED_PLANS", fakes.PlanNoId)
+			_, err := brokers.New(logger)
+			Expect(err).To(HaveOccurred())
+			os.Setenv("PRECONFIGURED_PLANS", fakes.PreconfiguredPlans)
```

Comment:
this shouldn't be in the test. I'm guessing you're doing it here because subsequent tests fail? if that's the case this should be getting set in a BeforeEach(..)